renderHtml.js

```diff
@@ -0,0 +1,74 @@
+export const renderHtml = (res) => {
+  const parentContainer = document.querySelector(".row");
+  parentContainer.innerHTML = "";
+
+  res.forEach((article) => {
+    const element = document.createElement("div");
+    element.classList = "mb-5";
+    const date = new Date(article.published_date);
+    const options = {
+      weekday: "long",
+      year: "numeric",
+      month: "long",
+      day: "numeric",
+    };
+    const correctDate = date.toLocaleDateString("en-US", options);
+
+    element.innerHTML = `
+    <div class="card pt-4 h-100" >
+      <img  src='${article.multimedia[1].url}' alt='${article.multimedia[1].type}'>
+      <div class="card-body mb-0">
+        <p class="card-title fs-6">${article.title}</p>
+        </div>
+          <div class="card-footer">
+              <small class="text-muted">Pulished on ${correctDate}</small>
+          </div>
+        </div>
+      </div>
+    </div>`;
+
+    parentContainer.appendChild(element);
+
+    const btn = document.createElement("button");
+    btn.innerHTML = "See more";
+    btn.classList = "btn px-3 mt-3 btn btn-outline-info";
+    element.appendChild(btn);
+
+    btn.addEventListener("click", () => {
+      detailedInformation(article);
+    });
+  });
+};
+
+const detailedInformation = (article) => {
+  clearDetails();
+  const detailedElement = document.createElement("div");
+
+  detailedElement.classList = "detailed";
+
+  detailedElement.innerHTML = `
```

Comment:
Let's display additional information right after the element, not at the bottom of the page. And let's display articles like cards here [cards example](https://i.imgur.com/CcdMjqV.png)